examples/audio.cpp

```diff
@@ -0,0 +1,49 @@
+#include <audio/synth.h>
+#include <q/support/notes.hpp>
+
+namespace q = cycfi::q;
+using namespace q::notes;
+
+int main(){
+
+    constexpr int triangle = 1;
+    constexpr int square   = 2;
+    constexpr int saw      = 3;
+
+    // 6 string guitar frequencies:
+    constexpr auto low_e   = E[2];
```

Comment:
Imagino que estes arranjos estejam em `q::notes`. Nesse caso, nem precisa trazer para o escopo local, colocar `q::notes::E` fica semanticamente mais claro mesmo. O ideal é dar `using namespace` só se formos usar muito mesmo, ou se for o namespace da nossa própria aplicação.